src/public/view/Invitation/Invitation.tsx

```diff
@@ -0,0 +1,72 @@
+import { Divider, Flex, Heading, Link } from '@adobe/react-spectrum';
+import Engagement from '@spectrum-icons/workflow/Engagement';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { createRoute } from '@tanstack/react-router';
+import ky from 'ky';
+
+import { useHref } from '../../../util/location';
+import { AuthView } from '../../component/AuthView';
+import { rootRoute } from '../../Root';
+import { loginRoute } from '../Login/Login';
+import { Credentials, InvitationForm } from './InvitationForm';
+
+export const invitationRoute = createRoute({
+  getParentRoute: () => rootRoute,
+  path: '/invitation/$code',
+  component: ClaimInvitation,
+});
+
+async function invitation(credentials: Credentials): Promise<any> {
+  const response = await ky
+    .post('/auth/invitation', {
```

Comment:
The backend exposes GraphQL API for claiming and accepting invitation. That should be used instead of GraphQL API.